features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/blocks/stats/api-throttledcounts/ajax/stats.jag

```diff
@@ -0,0 +1,168 @@
+<%
+include("/jagg/jagg.jag");
+var log = new Log();
+(function () {
+    response.contentType = "application/json; charset=UTF-8";
+    var mod, obj, result,
+            action = request.getParameter("action"),
+            apiName = request.getParameter("apiName"),
+            apiVersion = request.getParameter("apiVersion"),    
+            appName = request.getParameter("appName"),
+            fromDate = request.getParameter("fromDate"),
+            toDate = request.getParameter("toDate"),
+            site = require("/site/conf/site.json"),
+            msg = require("/site/conf/ui-messages.jag"),
+            allProviders = "__all_providers__";
+
+    convertToDateObj = function (date) {
```

Comment:
isn't it required to check for null and check for array length before proceeding?